src/test/java/uk/gov/justice/digital/delius/util/EntityHelper.java

```diff
@@ -554,6 +554,21 @@ public static PersonalCircumstance aPersonalCircumstance() {
             .build();
     }
 
+    public static PersonalCircumstance aPersonalCircumstance(final LocalDate startDate, final LocalDate endDate) {
+        return PersonalCircumstance.builder()
+            .personalCircumstanceId(1000L)
+            .offenderId(1001L)
+            .notes("Some notes")
+            .evidenced("Y")
+            .startDate(startDate)
+            .endDate(endDate)
+            .circumstanceType(CircumstanceType.builder().codeValue("CT").codeDescription("AP - Medication in Posession - Assessment").build())
+            .circumstanceSubType(CircumstanceSubType.builder().codeValue("CST").codeDescription("MiP approved").build())
+            .createdDatetime(LocalDateTime.of(2021, 7, 9, 9, 12))
+            .lastUpdatedDatetime(LocalDateTime.of(2021, 7, 9, 9, 32))
+            .build();
+    }
```

Comment:
DRY: change the existing `aPersonalCircumstance()` function to call this one with the static dates in use.

Reply:
Removed the duplication as recommended